app/controllers/items_controller.rb

```diff
@@ -6,14 +6,18 @@ def index
 
     @items = if @keyword.present?
       RWS::Ichiba::Item.
-        search(keyword: params[:keyword], imageFlag: 1).first(10)
+        search(keyword: params[:keyword], genre_id: @genre.id, imageFlag: 1).first(10)
     else
       []
     end
   end
 
   private
   def set_genres
-    @genre = RakutenWebService::Ichiba::Genre.root
+    if genre_id = params[:genre_id]
+      @genre = RakutenWebService::Ichiba::Genre.new(genre_id)
```

Comment:
__[rubocop]__
* Assignment in condition - you probably meant to use `==`.

<!-- Comment by [SideCI](https://sideci.com) -->